### PR TITLE
[Pipeline Refactor][Text-Generation] Create a helper function for creating engine_inputs

### PR DESCRIPTION
**Pull request by dbogunowicz**
The helper function is a useful abstraction, that makes the code more concise and allows for better testing.

Tested manually as well, by successfully running:
```python
from deepsparse.v2.text_generation.pipeline import TextGenerationPipeline


pipeline = TextGenerationPipeline(model_path="hf:mgoin/TinyStories-1M-deepsparse")
print(pipeline(prompt=["Who is the president of the USA?"]))
```
(the correctness was not checked though, only making sure the pipeline terminates without errors)

### CODE REVIEW

File: src/deepsparse/v2/text_generation/multi_engine_prefill_operator.py [hunk 1]
```diff
@@ -12,43 +12,24 @@
...
 
 
-class OnnxInputNames(Enum):
-    INPUT_IDS = "input_ids"
-    ATTN_MASK = "attention_mask"
-    CAUSAL_MASK = "causal_mask"
-    POSITIONS = "positions"
-
-
-# NOTE: A possible clean-up could involve combining this Operator and the
```
Comment (dbogunowicz) [inline src/deepsparse/v2/text_generation/multi_engine_prefill_operator.py:35]: Hopefully this PR resolves this todo note.

**Review by bfineran — APPROVED** [review]

**Review by dsikka — COMMENTED** [review]
This looks good - could you pull in my unit test PR and make sure the tests still pass?

**Comment by dbogunowicz** [comment]
@dsikka @bfineran Merged your tests in:

```bash
 pytest tests/deepsparse/v2/unit/
```

```bash
tests/deepsparse/v2/unit/text_generation/test_kv_cache.py .                                                                                                                                                                                                                                                      [ 11%]
tests/deepsparse/v2/unit/text_generation/test_msic.py .                                                                                                                                                                                                                                                          [ 22%]
tests/deepsparse/v2/unit/text_generation/test_process_inputs.py .                                                                                                                                                                                                                                                [ 33%]
tests/deepsparse/v2/unit/text_generation/test_single_token_engine.py ....                                                                                                                                                                                                                                        [ 77%]
tests/deepsparse/v2/unit/text_generation/test_token_generation.py ..                                                                                                                                                                                                                                             [100%]
```

**Review by bfineran — COMMENTED** [review]
fine after rebase

**Review by dsikka — APPROVED** [review]
LGTM. Will these be used in v1 as well?

**Pull request merged**